src/lib/operators/join_sort_merge/column_materializer.hpp

```diff
@@ -67,24 +93,51 @@ class ColumnMaterializer {
    **/
   std::shared_ptr<AbstractTask> _create_chunk_materialization_job(std::unique_ptr<MaterializedSegmentList<T>>& output,
                                                                   std::unique_ptr<PosList>& null_rows_output,
-                                                                  ChunkID chunk_id, std::shared_ptr<const Table> input,
-                                                                  ColumnID column_id) {
-    return std::make_shared<JobTask>([this, &output, &null_rows_output, input, column_id, chunk_id] {
+                                                                  const ChunkID chunk_id,
```

Comment:
If this is private, shouldnt it have an underscore? Why doesn't tidy complain?

Reply:
Can't follow? Function has an underscore. Or what are you looking at?

Reply:
nevermind